.github/workflows/performance.yml

```diff
@@ -178,74 +199,79 @@ jobs:
       - name: Run performance tests (current commit)
         run: npm run test:performance
 
-      - name: Print performance tests results
-        run: node ./tests/performance/results.js
-
-      - name: Check out target commit (target branch or previous commit)
+      - name: Get previous build (target branch or previous commit)
+        id: get-previous-build
         run: |
-          if [[ -z "$TARGET_REF" ]]; then
-            git fetch -n origin $TARGET_SHA
-          else
-            git fetch -n origin $TARGET_REF
-          fi
-          git reset --hard $TARGET_SHA
+          echo "artifact_id=$(gh api repos/WordPress/wordpress-develop/actions/artifacts?name="wordpress-build-$TARGET_SHA" --jq ".artifacts[] .id")" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up Node.js
-        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+      - name: Download artifact
```

Comment:
TODO:

There's a chance that the artifact does not exist yet. In that case, we have two options:

1. Exit early and ask to retry a few minutes later
2. Continue running tests, but simply without the comparison. Maybe add some explanation somewhere.

Reply:
can we trigger a retry directly? "does not exist yet" implies it will at some point, would that be in time for an immediate re-run?

Reply:
Probably not. Could take a few minutes in an edge case.

I think (2) is good enough and if not, one can just manually retry